webportal/src/app/market_list/components/item_list.jsx

```diff
@@ -1,16 +1,22 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 import React, { useState, useEffect } from 'react';
-import { Stack, Text, getTheme } from 'office-ui-fabric-react';
+import { Stack, Text, getTheme, ActionButton } from 'office-ui-fabric-react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
-import { capitalize, isNil } from 'lodash';
+import { capitalize, isNil, isEmpty } from 'lodash';
 import ItemCard from './item_card';
 import HorizontalLine from 'App/components/horizontal_line';
 import { listItems } from 'App/utils/marketplace_api';
 import Loading from 'App/components/loading';
 
-const { spacing } = getTheme();
+const { spacing, palette } = getTheme();
+
+const Section = styled.div`
```

Comment:
div` , is this a typo or an new expression?

Reply:
It is the standard syntax of styled component, please refer to [offcial example](https://styled-components.com/docs/basics#getting-started)